gluten-celeborn/common/src/main/java/org/apache/spark/shuffle/gluten/celeborn/CelebornShuffleManager.java

```diff
@@ -132,6 +142,11 @@ private SparkShuffleManager vanillaCelebornShuffleManager() {
     if (_vanillaCelebornShuffleManager == null) {
       synchronized (this) {
         if (_vanillaCelebornShuffleManager == null) {
+          if ("none"
+              .equalsIgnoreCase(
+                  conf.get(SPARK_CELEBORN_COMPRESSION_CODEC_KEY, celebornDefaultCodec))) {
+            conf.set(SPARK_CELEBORN_COMPRESSION_CODEC_KEY, celebornDefaultCodec);
```

Comment:
Shall we use a new spark conf copy to hold the configs for row-based celeborn shuffle manager ? I think it's risk to share the configs for both row-based and columnar shuffle manager, for example, the first shuffle is row-based then the conf will overwrite, and later the columnar shuffle manager would use the overwritten conf.